src/main/resources/db/migration/V1_65__elinks_alter_tables.sql

```diff
@@ -0,0 +1,40 @@
+--Alter dbjudicialdata.judicial_user_profile
+ALTER TABLE dbjudicialdata.judicial_user_profile ADD COLUMN deleted_flag boolean;
+ALTER TABLE dbjudicialdata.judicial_user_profile ADD COLUMN date_of_deletion TIMESTAMP;
+
+--Alter dbjudicialdata.judicial_office_appointment
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN base_location_id SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN appointment SET NOT NULL;
+ALTER TABLE dbjudicialdata.judicial_office_appointment ALTER COLUMN contract_type_id SET NOT NULL;
+
+--Alter dbjudicialdata.judicial_office_authorisation
```

Comment:
Please change the column name as mentioned in the ticket. It is juridiction only.

Reply:
Needful Done @manukundloo-hmcts